### ISSUE TITLE
medmod package "quadprog" incompatible with newer jamovi

### ISSUE THREAD

**Issue by HRDudeness** [raviselker/medmod#5]
In jamovi 0.9.23 (Linux flatpak), I cannot use medmod anymore:

Error : package ‘quadprog’ was installed by an R version with different internals; it needs to be reinstalled for use with this R version

Tried to remove und re-install medmod in jamovi. Without success.

**Comment by HRDudeness** [comment]
Addendum: medmod works with the Windows 0.9.23 without any issues. So, it seems to be a flatpak/Linux issue?

**Comment by raviselker** [comment]
Yeah, I think it's a flatpak issue. @jonathon-love can you fix this?

**Comment by jonathon-love** [comment]
oh yup.

**Comment by jonathon-love** [comment]
hi, i've pushed a new version of scatr up. remove the current version, then reinstall.

cheers

**Comment by raviselker** [comment]
Did you mean medmod?

**Comment by jonathon-love** [comment]
umm, no ... so i may not have done the right thing here.

**Comment by jonathon-love** [comment]
can you update medmod to the current `jas`?

**Comment by jonathon-love** [comment]
OK, uninstall and reinstall medmod, and you should be in business.

cheers

**Comment by raviselker** [comment]
@SvenHeimbuch does it work for you now?

**Comment by sjentsch** [comment]
@HRDudeness: Could you please check whether this is still a problem, and - if it is solved - close the issue? Thanks!

**Comment by raviselker** [comment]
I think it's fine to just close this after no activity for 8 years 😛
